src/main/java/org/candlepin/subscriptions/capacity/admin/InternalSubscriptionResource.java

```diff
@@ -83,14 +85,37 @@ public AwsUsageContext getAwsUsageContext(
             Usage.fromString(usage),
             BillingProvider.AWS,
             billingAccountId);
+
+    // Set start date one hour in past to pickup recently terminated subscriptions
+    var start = date.minusHours(1);
     List<Subscription> subscriptions =
         subscriptionSyncController.findSubscriptionsAndSyncIfNeeded(
-            accountNumber, Optional.empty(), usageKey, date, date, true);
+            accountNumber, Optional.empty(), usageKey, start, date, true);
+
+    var optionalTerminatedSubscription =
+        subscriptions.stream()
+            .filter(
+                subscription ->
+                    subscription.getEndDate().isBefore(date)
+                        && !subscription.getEndDate().isEqual(date))
+            .findFirst();
+
+    // Filter out any terminated subscriptions
+    var activeSubscriptions =
+        subscriptions.stream()
+            .filter(subscription -> subscription.getEndDate().isAfter(date))
```

Comment:
For consistency, this should be:
```suggestion
            .filter(subscription -> subscription.getEndDate().isAfter(date) || subscription.getEndDate().equals(date))
```

Reply:
isAfter and isBefore is are inclusive so it shouldn't be necessary in this filter:
```
  public boolean isAfter(OffsetDateTime other) {
  long thisEpochSec = this.toEpochSecond();
    long otherEpochSec = other.toEpochSecond();
    return thisEpochSec > otherEpochSec || thisEpochSec == otherEpochSec && this.toLocalTime().getNano() > other.toLocalTime().getNano();
  }
```

Reply:
`isAfter` and `isBefore` are not inclusive. Try
```
echo 'System.out.println(java.time.OffsetDateTime.parse("2022-01-01T00:00Z").isAfter(java.time.OffsetDateTime.parse("2022-01-01T00:00Z")))' | jshell -
```

What's subtle is that the method checks nanosecond values if the seconds are equal.

Given: time of now:

* `thisEpochSec = 1656424363`
* `otherEpochSec = 1656424363`
* `this.toLocalTime().getNano() = 515638000`
* `other.toLocalTime().getNano() = 515638000`

```
thisEpochSec > otherEpochSec || thisEpochSec == otherEpochSec && this.toLocalTime().getNano() > other.toLocalTime().getNano()
```
becomes
```
1656424363 > 1656424363 || 1656424363 == 1656424363 && 515638000 > 515638000
```
or
```
false || true && false
```
or
```
false
```

Reply:
Oh wow thanks for clarifying, will update this.